clients/drcachesim/common/trace_entry.h

```diff
@@ -736,12 +736,15 @@ typedef enum {
         OFFLINE_FILE_TYPE_ARCH_ARM32 | OFFLINE_FILE_TYPE_ARCH_X86_32 |
         OFFLINE_FILE_TYPE_ARCH_X86_64, /**< All possible architecture types. */
     /**
+     * Instruction addresses filtered online.
      * Note: this file type may transition to non-filtered. This transition is indicated
-     * by the TRACE_MARKER_TYPE_FILTER_ENDPOINT marker. Each window (which is indicated by
-     * the TRACE_MARKER_TYPE_WINDOW_ID marker) starts out filtered.
-     * TODO: add a new file type for mixed traces.
+     * by the #TRACE_MARKER_TYPE_FILTER_ENDPOINT marker. Each window (which is indicated
```

Comment:
I did add the namespace in drcachesim.dox.in to fix doxygen errors (https://github.com/DynamoRIO/dynamorio/pull/5717/commits/db0f65bf3414ce9f600a0ea44a38d18e22d3a9c1). I did not see doxygen errors here (probably since we are already "in" the namespace), but I have added it now for consistency.

Reply:
It does seem that doxygen sometimes figures out an unqualified name and sometimes seems to require fully qualified -- putting doxygen comments on the namespace declarations in the same file seems to help as I just found out yesterday: previously I thought it required fully qualified everywhere.  Given that, either way should be fine then; if doxygen accepts the unqualified for really long names maybe unqualified is nicer actually.